build.sbt

```diff
@@ -4063,9 +4063,6 @@ lazy val `engine-runner` = project
           `database-polyglot-root`
             .listFiles("*.jar")
             .map(_.getAbsolutePath()) ++
-          `google-polyglot-root`
```

Comment:
- The main change of this PR
- `std-bits/google` are no longer included when building `enso` NI